e2e/rust/build.rs

```diff
@@ -6,8 +6,9 @@ fn main() {
     builder.add({
         // Uppercase canister name is a workaround for using `ic-cdk-bindgen` with `dfx` >= 0.18.0
```

Comment:
@rvanasa : do we still need this file?

Reply:
Opened a PR with progress towards this: https://github.com/internet-computer-protocol/evm-rpc-canister/pull/295

Feel free to take over the PR and merge if it helps expedite the release process.

Reply:
I will overtake it as a separate PR. Thanks a lot for this!